src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -1061,7 +1061,7 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
         self ! InitialStartCall(callKey, CallActor.Start)
     }
 
-    if (backendCall.workflowDescriptor.readFromCache) {
+    def startCall = if (backendCall.workflowDescriptor.readFromCache) {
```

Comment:
This is one `if` expression, but I think our standard is to have braces around a multiline function and this is a lot of lines.